Project.toml

```diff
@@ -1,14 +1,15 @@
 name = "GenericLinearAlgebra"
 uuid = "14197337-ba66-59df-a3e3-ca00e7dcff7a"
-version = "0.2.7"
+version = "0.3.0"
```

Comment:
Why make this a breaking release? The compat bound change should be sufficient, no?